src/app/detail/ui/TaskEditor.tsx

```diff
@@ -165,7 +165,7 @@ export const TaskEditor = ({
           content={updateDetail}
           getContent={handleDetailChange}
           readonly={userType === UserType.CLIENT_USER}
-          editorClass="tapwrite-details-page"
+          editorClass=""
```

Comment:
Can we remove this altogether?

Reply:
solving this in another pr because editorClass is required prop for tapwrite.